src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -2675,6 +2687,27 @@ private void SetBackgroundImageBorder(Rectangle value)
             Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)SwigCPtr, View.Property.BACKGROUND, new PropertyValue(map));
         }
 
+        private void SetBorderlineColor(Color value)
+        {
+            if (value == null)
+            {
+                return;
+            }
+
+            (backgroundExtraData ?? (backgroundExtraData = new BackgroundExtraData())).BorderlineColor = value;
+
```

Comment:
'View.ApplyBorderline()' will use the value of 'backgroundExtraData'. Please remove 'PropertyMap map' usimg codes. we dont need them now.

Reply:
Thank you for your comment!
I have removed the PropertyMap code.